modules/Bio/EnsEMBL/Compara/PipeConfig/MergeDBsIntoRelease_conf.pm

```diff
@@ -99,6 +99,9 @@ sub default_options {
             # Mapping 'db_alias' => Arrayref of table names
             # Example:
             #   'family_db' => [qw(gene_member seq_member sequence)],
+            # Mapping 'db_alias' => Arrayref of table names
```

Comment:
This needs to be tidied up. You've duplicated the `Mapping 'db_alias'...` comment and the example is a bit defunct now that there are real entries.